modules/wikipedia/index.js

```diff
@@ -0,0 +1,27 @@
+const wiki = require('wikijs').default;
+
+module.exports.commands = ['wiki', 'wikipedia'];
+
+function prettyPage(page) {
+  return page.summary()
+    // Get the first "sentence" (hopefully)
+    .then(str => str.substr(0, str.indexOf('.') + 1))
+    // Truncate with an ellipsis if length exceeds 250 chars
+    .then(str => (str.length > 250 ? `${str.substr(0, 250)}...` : str))
+    // Append Wikipedia URL
+    .then(str => `${str} <${page.raw.canonicalurl}>`);
+}
+
+module.exports.run = function run(remainder, parts, reply, command) {
+  if (remainder.trim().length === 0) {
+    reply(`Usage: !${command} <search term(s)>`);
+    return;
```

Comment:
IMO commands with invalid inputs should silently fail instead of provide usage information.

Reply:
Done! Well it still isn't "silent", but I think it makes sense? Example above updated.